utils

```diff
@@ -60,11 +60,15 @@ function printTable(tableData)
   end
 end
 
-function formatInt(num)
+function formatInt(num, separator)
+  if separator == nil then
```

Comment:
condense into separator = separator or "," ?

Reply:
Used `or` syntax in the `else` block so it's only done if it is needed.